ui/apps/ui/src/app/dictionary/translateDictionaryValue.ts

```diff
@@ -74,7 +74,7 @@ export function translateDictionaryValue(
       return trainingDomainDictionary[valueType] || value;
       break;
     case DICTIONARY_TYPE_FOR_PIPE.GUIDELINE_PROVIDER:
-      return cleanGuidelineProvider(valueType);
+      return trainingDomainDictionary[valueType] || value; // cleanGuidelineProvider(valueType);
```

Comment:
If `cleanGuidelineProvider` is no longer required maybe the function itself should be removed?

Reply:
done